docs/CONTRIBUTING_DOCS.md

```diff
@@ -39,16 +39,14 @@ extensions = ['sphinx.ext.autodoc']
 
 Documentation is found in `docs/source`, and is divided into the following sections:
 
--  a **guide**, whose content needs to be
-written from scratch, 
-- **examples** which can be either jupyter notebooks or MyST formatted notebooks, and
-- an **API-doc** part, which is (mostly)
-automatically generated.
+-  a **guide**, whose content needs to be written from scratch for new features. For more details, go to [](contributing.md#proposing-a-new-feature-to-mitiq) and [](contributing_docs.md#adding-files-to-the-user-guide)
+- **examples** which can be either a jupyter notebook or a MyST formatted notebook. A checklist is available in [](contributing_docs.md#adding-example-files)
+- an **API-doc** part, which is (mostly) automatically generated. See [](contributing_docs.md#automatically-add-information-from-the-api-docs) for additional information.
 
 Information in the docs should be added as markdown files using the MyST markdown syntax.
-If you are adding a new file (as opposed to editing an existing one), ensure to add it to an associated TOC so that it is discoverable.
+If you are adding a new file (as opposed to editing an existing one), ensure to add it to an associated TOC so that it is discoverable. 
```

Comment:
```suggestion
If you are adding a new file (as opposed to editing an existing one), ensure to add it to an associated TOC so that it is discoverable.
```

We should set up a linter for this.

Reply:
If you use `make docs` without adding the new addition to the TOC, sphinx will complain that the file is not added anywhere.

Reply:
Sorry, I mean we should set up a linter to catch white space at the end of lines.

Reply:
Ah. Will have to search for something. 

I see there's a sphinx extension for it but a quick glance shows it is for `.rst` files only: https://github.com/sphinx-contrib/sphinx-lint